library/src/main/java/com/alexfu/countdownview/CountDownView.java

```diff
@@ -83,6 +84,13 @@ public void start() {
 
             @Override public void onFinish() {
                 stop();
+                currentDuration = 0L;
+                updateText(currentDuration);
+                invalidate();
```

Comment:
I'm curious as to why these three lines were added. When the timer is finished, the text is already zeroed out.

Reply:
No, it isn't zeroed out. The UI displays the last 1 second.

Reply:
@alfredayibonte hm, ok. This happens for me when the screen goes blank and I come back to it after the count down has finished. Is this what you were experiencing?

Reply:
No, my screen doesn't go blank. It simple updates the screen to that last 1 second. I think that's how the CountDownTimer works. The last tick happens in the onFinish method hence the reason why I'm updating the ui in the onFinish method.

Reply:
@alfredayibonte that's weird. That doesn't happen for me. What's the Android version you're running? My guess is that `CountDownTimer` on the platform version I'm testing on is slightly different than the platform version you're testing on. 

In any case, I would like this issue to be part of a separate PR since there needs to be more investigation and it's not really related to implementing callbacks.

Reply:
Cool, then i will take that bit out and create a seperate issue for that.

Reply:
Also I'm running on android 7.0 FYI